enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -28,6 +29,16 @@ export const FONT_SIZES = {
  * such as charts, data tables and popovers.
  */
 export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
+  collectionBrowser: {
+    breadcrumbs: {
+      expandButton: {
+        backgroundColor: "var(--mb-color-bg-light)",
+        hoverBackgroundColor: "var(--mb-color-brand)",
+        hoverTextColor: "var(--mb-color-text-white)",
+        textColor: "var(--mb-color-text-medium)",
```

Comment:
nit: Order this the same way we do in the readme
```suggestion
        textColor: "var(--mb-color-text-medium)",
        hoverTextColor: "var(--mb-color-text-white)",
```

Reply:
I went the other way and re-ordered the README